### DISCUSSION TITLE
Next.js 13, ts. Unexpected Video Element in URL Params from layout

### DISCUSSION THREAD

**Discussion by reikal1337** [vercel/next.js#56745] [Help] [answered]
### Summary

Page structure in my Next.js project: events/[eventId]

I have layout inside the events directory, which applies to both pages in events and events/[id]. Within the layout, I've included a simple video background. However, I've noticed that sometimes when I navigate from the event page to the event/[eventId] page, the event[id] page in the URL parameters receives a element with a src or poster attribute. I'm not sure why this is happening or how to fix it.

Any help or insights on how to resolve this issue would be greatly appreciated!

events/layout.tsx:
```
export default function RootLayout({
  children,
}: {
  children: React.ReactNode
}) {
  return (
    <>
        <BackgroundVideo />
            {children}
    </>
    
  );
}
```

BackgrounVideo component:
```
"use client"
const BackgroundVideo = () => {
  const [isClient, setIsClient] = useState(false);

  useEffect(() => {
    setIsClient(true)
  },[])

  return(
    <div >
      {
      isClient && 
      <video 
        autoPlay loop muted
        poster="videoBgPoster.jpg"
        >
          
      <source src="videoBg.mp4" 
              type="video/mp4"/>
      </video>
      }
      <div className="z-[-90] fixed bg-black bg-opacity-40 h-full w-full"></div>
    </div>
  );
}

export default BackgroundVideo;
```

events/[eventId]:
```
type Props = {
  params: {
    eventId: string,
  },
  searchParams: Record<string, string> | null | undefined
}

const Event = async ({ params, searchParams }: Props) => {

  console.log("Params...: ", params)
}
```

### Additional information

```js
Sometimes it receives proper id, sometimes this...
Console:
Params...: { id: 'videoBgPoster.jpg' }
Params...: { id: 'videoBg.mp4' }
```

**Comment by icyJoseph** [comment, marked as answer]
I guess the problem is the way the video src paths are written, try giving them absolute paths like `/videoBgPoster` or the like
